src/drops/HolographNFTCreatorProxy.sol

```diff
@@ -1,9 +1,61 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.13;
+/*HOLOGRAPH_LICENSE_HEADER*/
 
-import {ERC1967Proxy} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+/*SOLIDITY_COMPILER_VERSION*/
 
-/// @dev Holograph NFT Creator Proxy Access Contract
-contract HolographNFTCreatorProxy is ERC1967Proxy {
-  constructor(address _logic, bytes memory _data) payable ERC1967Proxy(_logic, _data) {}
+import "../abstract/Admin.sol";
```

Comment:
what's is `src/drops/HolographNFTCreatorProxy.sol` vs `src/drops/ERC721DropProxy.sol`

Reply:
oh yeah i see this is for the `src/drops/HolographNFTCreatorV1.sol` instead